scripts/setup_database.php

```diff
@@ -27,6 +27,39 @@
 
 // NOTE: these sql commands could be db agnostic if we change the blobs to text
 
+// Check if the required extensions for the configured DB driver are loaded
+if ($db_driver == 'mysql') {
+    $required_extensions = ['mysqli', 'mysqlnd', 'pdo_mysql'];
```

Comment:
@Baraka24 can you please create a function to remove duplicate codes share by these 2 if statements ?

Reply:
[PR](https://github.com/cypht-org/cypht/pull/1390)